redis/client.py

```diff
@@ -707,6 +707,12 @@ def __init__(self, host='localhost', port=6379,
     def __repr__(self):
         return "%s<%s>" % (type(self).__name__, repr(self.connection_pool))
 
+    def __eq__(self, other):
+        return (
```

Comment:
Do you think it's better written as-is, or using duck typing as so?

    try:
        return self.connection_pool == other.connection_pool
    except AttributeError:
        return False